rmw_connext_cpp/src/functions.cpp

```diff
@@ -934,6 +934,47 @@ rmw_create_client(
     goto fail;
   }
 
+  // For now, use the default QoS profile for services.
```

Comment:
No problem, we can split work and I can deal with `rclcpp` if you're ok with that.